test/integration/integration_test.cc

```diff
@@ -107,6 +109,13 @@ TEST_P(IntegrationTest, AdminDrainDrainsListeners) {
 
   upstream_request_->encodeHeaders(default_response_headers_, false);
 
+  EXPECT_THROW_WITH_REGEX(
```

Comment:
nit: This may not be required as we know it is in use as we are making connection to that port

Reply:
+1 I would just remove this.